src/components/PlayWord/PlayWord.jsx

```diff
@@ -19,7 +20,15 @@ export class PlayWord extends React.Component {
     }
 
     sendMessage() {
-        this.props.playWord(this.state.currentValue);
+        axios.post('/api/validWord', {
+            word: this.state.currentValue
+        })
+        .then(() => {
+            this.props.playWord(this.state.currentValue);
```

Comment:
So I'm thing this should move into the actions. You'd call an action with `this.state.currentValue` and update the state from there.